bin.ts

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env node
 
 import { QuickImportCycleChecker } from './src/quick-import-cycle-checker';
+import { parseArgs } from './src/util/parse-arguments.util';
 
-// TODO Make this more elegant/configurable
-const directoryToCheck = process.cwd();
-const rootDirectory = process.cwd();
+export interface cliSettings {
+    checkDirectories: string[];
+    root: string;
```

Comment:
root is allowed to be nullish I think, can be omitted

Reply:
If no argument is passed it will default to process.cwd()

Reply:
Since we give it a default value I don't think allowing null is